app/index.html

```diff
@@ -33,6 +33,7 @@
         <script src="components/angular-resource/angular-resource.js"></script>
         <script src="components/angular-spotify/src/angular-spotify.js"></script>
         <script src="components/angular-socket-io/socket.js"></script>
+        <script src="components/angular-debounce/dist/angular-debounce.js"></script>
 
```

Comment:
I tried that but it didn't seem to work with the slider as it debounces the update of ng-model which the slider relies on to adjust the display of the slider handler. We won't need this anyway if we're moving to buttons.